app/src/App.css

```diff
@@ -1,3 +1,78 @@
+.dashboard {
```

Comment:
Can you move this CSS into its own file? We will use `App.css` as a top level CSS file

something like `/app/assets/stylesheets/`

Reply:
surely!